specs/activity-api.yaml

```diff
@@ -383,6 +385,10 @@ paths:
               schema:
                 $ref: 'common.yaml#/schemas/ErrorMessage'
 components:
+  securitySchemes:
```

Comment:
What do you think of moving it to `common.yaml` as it is same for all the APIs?

Reply:
Good idea!

Tried to reference both the `security` and the `securitySchemes`, but only the latter worked.

Updated the PR.